modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -73,6 +73,9 @@ This section provides the necessary details that enable you to control egress tr
 |Provides core container images as a fallback when quay.io is not available.
 |===
 +
+[NOTE]
```

Comment:
This admonition should include the ==== format as seen in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#admonitions